userspace/libsinsp/container.cpp

```diff
@@ -82,195 +82,300 @@ const sinsp_container_info::container_mount_info *sinsp_container_info::mount_by
 	return NULL;
 }
 
-sinsp_container_manager::sinsp_container_manager(sinsp* inspector) :
-	m_inspector(inspector),
-	m_last_flush_time_ns(0)
+#ifdef CYGWING_AGENT
```

Comment:
Unless I'm missing something, this ifdef closes at line 876, but in the meantime CYGWING_AGENT is used multiple times. Is that intended?

Reply:
good point

Reply:
I fixed what I could but I wasn't able to build sysdig or agent on Windows so I can't really say if it's going to work. Will push once the tests pass.